include/nlohmann/json.hpp

```diff
@@ -8333,8 +8334,8 @@ class basic_json
             }
 
             // collect mandatory members
-            const std::string op = get_value("op", "op", true);
-            const std::string path = get_value(op, "path", true);
+            const std::string op(get_value("op", "op", true));
```

Comment:
Is there a technical reason for this change (3 lines)?

Reply:
The previous makes use of implicit conversions, which does not build when using the `explicit` operator.

Reply:
Ah, fun with different initialization syntaxes.